Userland/Utilities/icc.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2022, Nico Weber <thakis@chromium.org>
+ * Copyright (c) 2023, Nico Weber <thakis@chromium.org>
```

Comment:
"bump" was unclear, sorry - `2022-2023` is what we usually do

Reply:
The good news is that I'll only bother you again next year! :^)

Reply:
We're gonna stop updating copyright years in 2024?! That's _great_ news!

(Done, and threw in a tiny second cleanup commit for free too.)